### ISSUE TITLE
Get proper repo link from package.json or from upstream repo

### ISSUE THREAD

**Issue by alexindigo** [greenkeeperio/greenkeeper#131]
Tried to enable greenkeeper on one of the projects I'm maintaining:

```shell
$ gk enable
🌴  info enable The GitHub slug is: alexindigo/form-data
alexindigo/form-data enabled
$ git remote -v
origin	git@github.com:alexindigo/form-data.git (fetch)
origin	git@github.com:alexindigo/form-data.git (push)
upstream	git@github.com:form-data/form-data.git (fetch)
upstream	git@github.com:form-data/form-data.git (push)
```

Obviously I'd like to see PRs (and checks) against upstream repo,
and I don't see any related option I add run `gk` with.

And wording itself doesn't convey what to expect.
`enable Greenkeeper for a package`, it doesn't say for your github repo or for your personal fork,
it said for "package".

Thanks.

**Comment by christophwitzko** [comment]
Hi @alexindigo,

 `greenkeeper enable` always tries to enable the repository that is set in the git remote named origin.
To enable an independent repository you can use the `--slug` flag. (e.g. `greenkeeper enable --slug form-data/form-data`)

You are right, the wording is not the best. We will fix that. :+1: 

Have a nice day,
Christoph

**Comment by alexindigo** [comment]
Wouldn't be more logical to get repo from the `package.json` or is there anything else in pla I'm not aware of? Thank you.

**Comment by christophwitzko** [comment]
The wording is now fixed in #134.
It's more likely users have admin access (Greenkeeper needs admin access to work correctly) to the repository that is set in the git remote named origin.
